js/module.js

```diff
@@ -415,3 +415,7 @@ Module.register = function (name, moduleDefinition) {
 	Log.log("Module registered: " + name);
 	Module.definitions[name] = moduleDefinition;
 };
+
+exports._test = {
```

Comment:
This isn't work :(. I sent a new Pull Request with a hotfix. Can you review @nhubbard and @qistoph this?

Also, I add a post for create a set list of test for the suite. 
https://forum.magicmirror.builders/topic/1456/test-suite-for-magicmirror
I would really like have your comments.